packages/@aws-cdk/aws-appsync/lib/graphqlapi-base.ts

```diff
@@ -0,0 +1,193 @@
+import { ITable } from '@aws-cdk/aws-dynamodb';
+import { IFunction } from '@aws-cdk/aws-lambda';
+import { CfnResource, IResource, Resource } from '@aws-cdk/core';
+import { DynamoDbDataSource, HttpDataSource, LambdaDataSource, NoneDataSource } from './data-source';
+
+/**
+ * Optional configuration for data sources
+ */
+export interface DataSourceOptions {
+  /**
+   * The name of the data source, overrides the id given by cdk
+   *
+   * @default - the id provisioned
+   */
+  readonly name?: string;
+
+  /**
+   * The description of the data source
+   *
+   * @default - No description
+   */
+  readonly description?: string;
+}
+
+/**
+ * Interface for GraphQL
+ */
+export interface IGraphqlApi extends IResource {
+
+  /**
+   * an unique AWS AppSync GraphQL API identifier
+   * i.e. 'lxz775lwdrgcndgz3nurvac7oa'
+   *
+   * @attribute
+   */
+  readonly apiId: string;
+
+  /**
+   * the ARN of the API
+   *
+   * @attribute
+   */
+  readonly arn: string;
+
+  /**
+   * add a new dummy data source to this API. Useful for pipeline resolvers
+   * and for backend changes that don't require a data source.
+   *
+   * @param id The data source's id
+   * @param options The optional configuration for this data source
+   * @default name - the id
+   * description - No description
+   */
+  addNoneDataSource(id: string, options?: DataSourceOptions): NoneDataSource;
+
+  /**
+   * add a new DynamoDB data source to this API
+   *
+   * @param id The data source's id
+   * @param table The DynamoDB table backing this data source
+   * @param options The optional configuration for this data source
+   * @default name - the id
+   * description - No description
```

Comment:
I believe this should be indented by a couple spaces so that it's clear that this comment belongs to the `@default` 
we don't have too many examples that span beyond a single line, but that's the guidance I was given in the past :)

Reply:
im just removing them